docker-compose.yml

```diff
@@ -7,6 +7,14 @@ services:
     volumes:
       - 'data:/data'
     network_mode: host
+    environment:
+      - 'ROUTER_INTERFACE=eth0'
+      - 'ROUTED_PREFIX=001:db8:1::/64'
+      - 'TUNNEL_PREFIX=001:db8:2::/64'
+      - 'TUNNEL_REMOTE_IP4=203.0.113.1'
+      - 'TUNNEL_LOCAL_IP4=192.0.2.1'
+      - 'TUNNEL_MTU=1480'
+      - 'CLIENTS_WHITELIST=001:db8:1::'
```

Comment:
Nice! Re CLIENTS_WHITELIST, doesn't the value have to start with `fe80::` for `radvd`? (Consider that before the client obtains a global IPv6 address, it will only have a link local address.) Btw, these sample values starting with `001:`, that's the same as starting with `1:`. Perhaps you meant starting with `2001:`? (Copy and paste issue?)  And if these values are your real values, consider whether we want to commit them to the repo for posterity. Maybe more "obviously fake" values would be better, e.g. `2001:cafe:1234:1::/64` and `fe80::f00:bee:baba:3344`?

Reply:
hi @pdcastro I am purposedly using documentation prefixes as described in [RFC3849](https://tools.ietf.org/html/rfc3849) and [RFC5737](https://datatracker.ietf.org/doc/html/rfc5737). These won't work but they should also not cause trouble I believe. Each user will need to customize them.

Reply:
The RFC mentions prefix `2001:`, but above the entries start with `001:`.

Reply:
you're right, that was a global replacement mistype. It's fixed now.